daft/logical/logical_plan.py

```diff
@@ -493,7 +480,7 @@ def __repr__(self) -> str:
         )
 
     def required_columns(self) -> list[set[str]]:
-        return [{self._filepaths_column_name} | self._predicate.required_columns()]
+        return [{"file_paths"} | self._predicate.required_columns()]
```

Comment:
Hmm it appears that it is not: the only place that it could happen is in the `(Projection, LogicalPlan)` optimization rule, and that rule doesn't apply to `TabularFileScan` nodes: https://github.com/Eventual-Inc/Daft/blob/102de0f7567ede08f21bf810779a1834755a0401/daft/logical/optimizer.py#L219-L220